src/pocketmine/Player.php

```diff
@@ -1378,7 +1385,7 @@ protected function checkNearEntities($tickDiff){
 	}
 
 	protected function processMovement($tickDiff){
-		if(!$this->isAlive() or !$this->spawned or $this->newPosition === null or $this->teleportPosition !== null or $this->isSleeping()){
```

Comment:
Wouldn't this be considered [anticheat](https://github.com/pmmp/PocketMine-MP/projects/2)?

If there is a better way to implement this PR, I would be glad to.  I tried implementing what was previously suggested.  I might need someone to give me a push in the right direction.